### ISSUE TITLE
Prevent duplicate locations when adding bathrooms

### ISSUE THREAD

**Issue by KurtPrice** [RefugeRestrooms/refugerestrooms#399]
Currently, duplicate bathrooms can be submitted due to very small differences in addresses.
For example: In Bellingham, Wa the restroom in The Black Drop Coffeehouse appears twice because one address is listed as
300 West Champion
and the other restroom as
300 W Champion
There is only one restroom at this location, but it is listed as two separate restrooms, with two separate ratings because of the difference of W vs. West.

Further complications are made by BELLINGHAM vs. Bellingham and WA vs. Washington

These should be a simple fix by reducing all city names to lowercase for comparison and string parsing for W, E, N, S, etc. to their corresponding West, East, North, South, etc.

**Comment by Arinzeokeke** [comment]
Yeah, it should ignore case sensitivity for data uniqueness. On the other issue, the workable solution will be to show a confirmation prompt if a new bathroom is too similar to an already saved one.
